setup.py

```diff
@@ -14,7 +14,7 @@
 
 setup_requirements = []
 
-test_requirements = []
+test_requirements = ['ddt', 'pytest']
```

Comment:
why re-add pytest?

Reply:
re-add? It only appear once in setup.y

Reply:
I removed it in the previous commit:

https://github.com/SciQLop/broni/commit/e304b39fd4e88c304a3ec66cda541e4cdb8e1a13#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L17

Reply:
I see, I missed this one. The issue is that ideally both setup.py and requirements_dev.txt should be usable.
Developer should use requirements_dev.txt while end user should be able to run ```python setup.py test```.

Reply:
I just oversaw that requirements_dev.txt contained pytest.

Could we not load the file in setup.py to avoid the redundancy?

Reply:
Some people do that but it defeat the purpose of each https://packaging.python.org/discussions/install-requires-vs-requirements/#requirements-files